runtime/near-vm-runner/tests/test_error_cases.rs

```diff
@@ -277,15 +277,29 @@ fn stack_overflow() -> Vec<u8> {
 
 #[test]
 fn test_stack_overflow() {
-    // We only test trapping tests on Wasmer, as of version 0.17, when tests executed in parallel,
-    // Wasmer signal handlers may catch signals thrown from the Wasmtime, and produce fake failing tests.
-    assert_eq!(
-        make_simple_contract_call_vm(&stack_overflow(), b"hello", VMKind::Wasmer),
-        (
-            Some(vm_outcome_with_gas(63226248177)),
-            Some(VMError::FunctionCallError(FunctionCallError::WasmUnknownError))
-        )
-    );
+    with_vm_variants(|vm_kind: VMKind| {
+        // We only test trapping tests on Wasmer, as of version 0.17, when tests executed in parallel,
+        // Wasmer signal handlers may catch signals thrown from the Wasmtime, and produce fake failing tests.
```

Comment:
Is it safe to do parallel testing of Wasmer 0.17 and Wasmer 1.0 given the context this comment provides? I wonder if we have any protection in our regular (non-testing) context against parallel execution of VMs.

Reply:
It is safe, as the comment said wasmtime signals would be falsely caught by wasmer, this is due to specialty of wasmtime throw signals. Also it's not this test failed, it's other test execute in parallel incorrectly catch wasmtime signals. So just avoid run wasmtime signal test in this test case will not break our test